10376.xml

```diff
@@ -0,0 +1,202 @@
+INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+POSSIBILITY OF SUCH DAMAGE.
+
+-->
+<LWM2M xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://www.openmobilealliance.org/tech/profiles/LWM2M.xsd">
+	<Object ObjectType="MODefinition">
+		<Name>Periodic Activity</Name>
+		<Description1>
+		<![CDATA[Generic interface to set up periodic activity like hourly sensor reading/load activation and mechanism to retrieve the periodic data.]]>
+		</Description1>
+		<ObjectID>10376</ObjectID>
+		<ObjectURN>urn:oma:lwm2m:x:10376</ObjectURN>
+		<LWM2MVersion>1.0</LWM2MVersion>
+		<ObjectVersion>1.0</ObjectVersion>
+		<MultipleInstances>Multiple</MultipleInstances>
+		<Mandatory>Optional</Mandatory>
+		<Resources>
+			<Item ID="0">
+				<Name>Activity Name</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Mandatory</Mandatory>
+				<Type>String</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units></Units>
+				<Description><![CDATA[This resource represents activity's name or string reference declared by device. It cannot be modified externally. Device can support multiple activities by declaring multiple instances of Periodic Activity object.]]>
+				</Description>
+			</Item>
+			<Item ID="1">
+				<Name>Activity Description</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Mandatory</Mandatory>
+				<Type>String</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units></Units>
+				<Description><![CDATA[This resource describes activity's operation including it's supported options and data returned.]]>
+				</Description>
+			</Item>
+			<Item ID="2">
+				<Name>Activity Settings</Name>
+				<Operations>RW</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Opaque</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units></Units>
+				<Description><![CDATA[This resource allows activity to accept settings or input parameters when run. The available settings, if supported, should be described using Activity Description resource.]]>
+				</Description>
+			</Item>
+			<Item ID="3">
+				<Name>Start Mask</Name>
+				<Operations>RW</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Mandatory</Mandatory>
+				<Type>String</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units></Units>
+				<Description><![CDATA[This resource defines when to start the periodic activity. Start Mask needs to adhere to following string format:
```

Comment:
The description field has many tab characters that may not work well with all UIs. I'd recommend using only line changes. Same comment applies to many other descriptions here.

Reply:
I edit the xml using notepad++. I have removed all the tabs.